src/app/parcels/ActionBar/ActionModals/DeleteParcelModal.tsx

```diff
@@ -49,36 +50,27 @@ const DeleteParcelModalContent: React.FC<ContentProps> = ({
                 maxParcelsToShow={maxParcelsToShow}
             />
             <ConfirmButtons>
-                <Button variant="contained" onClick={onClose}>
-                    Cancel
-                </Button>
-                <Button
-                    variant="contained"
+                <ButtonWrap>
+                    <Button variant="outlined" onClick={onClose}>
+                        Cancel
+                    </Button>
+                </ButtonWrap>
+                <DeleteButton
                     onClick={() => setIsConfirmationDialogueOpen(true)}
                     ref={modalDeleteButtonFocusRef}
                 >
                     Delete
-                </Button>
+                </DeleteButton>
             </ConfirmButtons>
-            <Dialog
-                open={isConfirmationDialogueOpen}
+            <DeleteConfirmationDialog
```

Comment:
would it be possible for cancel to be autofocused, not delete? I think having all autofocuses on delete throughout all modals/dialogs means that a user can delete something without really paying attention to the dialog message.